Client/Frontend/Home/FirefoxHomeViewModel.swift

```diff
@@ -87,11 +87,7 @@ class FirefoxHomeViewModel: FeatureFlagsProtocol {
         // For Pocket, the user preference check returns a user preference if it exists in
         // UserDefaults, and, if it does not, it will return a default preference based on
         // a (nimbus pocket section enabled && Pocket.isLocaleSupported) check
-        guard featureFlags.isFeatureActiveForBuild(.pocket),
-              featureFlags.userPreferenceFor(.pocket) == UserFeaturePreference.enabled
-        else { return false }
-
-        return true
+        return featureFlags.isFeatureBuildAndUserEnabled(.pocket)
```

Comment:
Is this change necessary now for this PR? Just asking since I'm currently refactoring the viewModels in home page, and would benefit of removing this conflict. I can enable this change in my PR.

Reply:
sure I can remove this. Is it just this change in `FirefoxHomeViewModel`? OR anywhere else?

Reply:
Some in history view model, but I can deal with that.